app/serializers/v1/fee_serializer.rb

```diff
@@ -19,6 +21,8 @@ def serialize
         total_amount_currency: model.amount_currency,
         units: model.units,
         events_count: model.events_count,
+        invoice_id: model.invoice_id,
```

Comment:
`lago_invoice_id`? (not really fan of prepending by `lago_` but that's what we currently have for other ids, no?)

Reply:
For me, the preferable solution would be to prepend by `external_` each time it's not related to a Lago id...

Then, no need to prepend by `lago_` each time... something maybe to think about in a future breaking change refactor 😅